app.js

```diff
@@ -0,0 +1,98 @@
+const express = require("express");
+const cors = require("cors");
+const bodyParser = require("body-parser")
+const timestamp = require('time-stamp');// ex . timestamp: req.timestamp
+const School = require('./School')
+const Student = require('./Student')
+
+const port = 3000;
+
+const app = express();
+app.use(cors());
+
+app.use(bodyParser.urlencoded({extended: false}));
+app.use(bodyParser.json());
+
+
+let myschool = new School();
+
+const enrollStu = (req,res, next) => {
+    let studSample = new Student(req.body.name, req.body.age, req.body.city, req.body.grade)
+    let students = myschool.enrollStudent(req.params.className.toLocaleUpperCase(), studSample)
+    if(students){
+        res.json({ 
+            "student": students,
+            "className": req.params.className.toLocaleUpperCase(),
+            "message": "Enrolled Student",
+            "timestamp": timestamp.utc('YYYY, MM/DD HH:mm:ss')
+          })
+        console.log("Middleware A has been fired");
+    } else {
+        res.json({ 
+            "error": "Please fill out all the information for the student",
+            "timestamp": timestamp.utc('YYYY, MM/DD HH:mm:ss')
+          })
+    }
+}
+
+const getStudents = (req,res, next) => {
+    let className = req.params.className.toUpperCase()
+    console.log(req.body)
+    let sClass = myschool.getStudentsByClassWithFilter(className, req.body.failing, req.body.city)
+
+    if(sClass){
+        res.json({
+            "students": sClass,
+            "message": "Retrieved Students",
+            "timestamp": timestamp.utc('YYYY, MM/DD HH:mm:ss')
+          })
+    } else {
+        res.json({ 
+            "error": `Class ${className} doesn't exist.`,
+            "timestamp": timestamp.utc('YYYY, MM/DD HH:mm:ss')
+          })
+    }
+}
+
+app.get("/", (req, res) => {
+    console.log(myschool.classes)
+    res.json({Message : "Hello Welcome to Jay School ",
+               Classes:  myschool.classes
+                })
+    
+})
+app.post("/class/:className/enroll/", enrollStu, (req, res) => {
```

Comment:
Since in your `enrollStu` already returns a response already, so here for your last callback function, you will get an error. You can just exclude it.